eZ/Bundle/EzPublishCoreBundle/Resources/views/default/embed.html.twig

```diff
@@ -0,0 +1,3 @@
+{% for field in content.FieldsByLanguage %}
+    {{ ez_render_field(content, field.fieldDefIdentifier) }}
+{% endfor %}
```

Comment:
ok. and so, even if i had my own bundle, those core bundles template files will be used if i don't override them?